recipes/molbloom/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "molbloom" %}
+{% set version = "2.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/whitead/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 4fb187a612f8da59473ac78b893d94fb6c1c6c2daf4027a8228e8b9dc00e7b13
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+
+  host:
+    - python >=3.6
+    - pip
+    - cython
+  run:
+    - python >=3.6
+    - rdkit
+    - importlib_resources
+
+test:
```

Comment:
```suggestion
test:
  requires:
    - pip
  commands:
    - pip check
```

Reply:
@xhochy pip check doesn't seem to work due to the way compiled extensions are imported  within the package, are we able to skip this bit?

Reply:
`pip` check doesn't execute the python code at all but simply checks metadata consistency. So imports shouldn't affect it.

Reply:
Lets try, perhaps you can see what is going on, bit bamboozling to me.